use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -1,46 +1,222 @@
 ---
-title: Enabling the object storage tier
-excerpt: How to enable the object storage tier
+title: Manage tiering
+excerpt: How to enable and use object storage tiering
 products: [cloud]
 keywords: [tiered storage]
 tags: [storage, data management]
 cloud_ui:
     path:
         - [services, :serviceId, overview]
+plans: [scale, enterprise]  
 ---
 
-# Tier data to the object storage tier
+# Manage tiering
 
-Enable tiered storage to begin migrating rarely used data from Timescale's standard high-performance storage tier
-to the object storage tier to save on storage costs. 
+You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. After you enable tiered storage, you then either create automated tiering policies or manually tier and untier data.
 
-## Enabling the object storage tier
+<Highlight type="note">
```

Comment:
Can you move this to under `When tiered storage is activated` please

Reply:
moved. does not make sense to me to give the steps first and only explain why it does not work as an afterthought